source/2_volplugin.md

```diff
@@ -19,7 +19,7 @@ Clone and build the project:
 
 #### Everywhere else (with a VM):
 
-* `git clone github.com/contiv/volplugin`
+* `git clone https://github.com/contiv/volplugin.git`
```

Comment:
oh, I have always used a ".git". But just checked, that it works without it too. I will remove the ".git" Thanks.